test/blackbox-tests/test-cases/install-dir/install-datadir.t/run.t

```diff
@@ -0,0 +1,14 @@
+  $ echo "(lang dune 2.0)" > dune-project
```

Comment:
This test needs a description. By the way, you could rename it to `install-datadir.t` and remove the extra directory as well.